storage/config_test.go

```diff
@@ -31,3 +33,15 @@ func TestSetDefaults(t *testing.T) {
 		})
 	}
 }
+
+func TestConfiguration_GetSpansArchiveTable(t *testing.T) {
+	const repetitionCount = 100
+	defaultConfig := Configuration{}
+	defaultConfig.setDefaults()
+	assert.Equal(t, defaultSpansTable+"_archive", defaultConfig.getSpansArchiveTable())
+	for i := 0; i < repetitionCount; i++ {
+		tableName := "table_" + strconv.FormatUint(rand.Uint64(), 16)
```

Comment:
avoid using random in tests. The tests should be repeatable and predictable

Reply:
It's pseudo-random actually, so they are predictable. But yeah, I think I should get rid of random here.